src/components/wms/InformationPanel.vue

```diff
@@ -0,0 +1,201 @@
+                @blur="changecolorScaleRange"
+                :rules="[
+                  rules.required,
+                  rules.biggerThanZero,
+                  rules.smallerThanMax,
+                ]"
+              ></v-text-field>
+            </v-col>
+            <v-col cols="6">
+              <v-text-field
+                v-model.number="mutableColorScaleRange.max"
+                label="Max"
+                outlined
+                variant="underlined"
+                hide-details
+                @keydown.enter.stop="changecolorScaleRange"
+                @blur="changecolorScaleRange"
+                :rules="[rules.required, rules.biggerThanMin]"
+              ></v-text-field>
+            </v-col>
+          </v-row>
+        </v-list-item>
+      </v-list-group>
+      <v-list-item v-if="props.completelyMissing">
+        Wms layer is completely missing
+      </v-list-item>
+    </v-list>
+  </v-menu>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { DateTime } from 'luxon'
+import { Style } from '@deltares/fews-wms-requests'
+import { ref } from 'vue'
+
+interface Props {
+  layerTitle: string
+  currentTime?: Date
+  forecastTime?: Date
+  styles?: Style[]
+  completelyMissing?: boolean
+  firstValueTime?: Date
+  lastValueTime?: Date
+  colorScaleRange?: { min: number; max: number }
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  layerTitle: '',
+})
+
+const emit = defineEmits(['style-click', 'color-scale-range-change'])
+const layersIcon = 'mdi-layers'
+const timeIcon = 'mdi-clock-time-four-outline'
+const rangeIcon = 'mdi-layers-edit'
+const colorScalesIcon = 'mdi-palette'
+const selectedStyle = ref<Style>()
+const mutableColorScaleRange = ref(props.colorScaleRange)
+
+const rules = {
+  required: (v: number) =>
+    (v !== null && v !== undefined) || 'Field is required',
+  biggerThanZero: (v: number) =>
+    v >= 0 || 'Value must be bigger or equal than 0',
+  smallerThanMax: (v: number) =>
+    (mutableColorScaleRange.value && v < mutableColorScaleRange.value.max) ||
+    'Value must be smaller than max',
+  biggerThanMin: (v: number) =>
+    (mutableColorScaleRange.value && v > mutableColorScaleRange.value.min) ||
+    'Value must be bigger than min',
+}
+
+const analysisTime = computed(() => {
+  if (!props.forecastTime) return 'Analysis time not available'
+  return (
+    'Analysis time: ' +
+    DateTime.fromJSDate(props.forecastTime).toFormat('dd/MM/yyyy, HH:mm:ss')
+  )
+})
+
+const formattedTimeRange = computed(() => {
+  if (!props.firstValueTime || !props.lastValueTime) return ''
+  const format = 'dd/MM/yyyy, HH:mm:ss'
+  return `${DateTime.fromJSDate(props.firstValueTime).toFormat(
+    format,
+  )} → ${DateTime.fromJSDate(props.lastValueTime).toFormat(format)}`
+})
+const formattedCurrentTime = computed(() => {
+  if (!props.currentTime) return ''
+  const format = 'HH:mm ZZZZ'
+  const timeZone = 'Europe/Amsterdam'
```

Comment:
Do we always want nl / amsterdam here?
WebOC is used quite broadly, maybe get this from the users locale.